index.js

```diff
@@ -9,13 +9,20 @@ var stripJs = function(htmlContent) {
    }
 
    // Sanitize the HTML first:
+   var doctypeRegex = /<!DOCTYPE .*?>/i;
```

Comment:
Can you change the regex to the following: `/^\s*<!DOCTYPE .*?>/i`
DOCTYPE is supposed to be the very first thing in an HTML document, so the regex above will take care of that.